environments/Test-Laptop.json

```diff
@@ -41,6 +41,9 @@
         "hdp_utils": "http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12"
       },
       "hadoop" : {
+        "environment" : {
+          "profile" : ""
```

Comment:
@pu239ppy since this new attribute is for users to include logic in a ``chef`` wrapper cookbook, users can choose any data structure they would need to build the logic in the wrapper cookbook.